### DISCUSSION TITLE
View Background

### DISCUSSION THREAD

**Discussion by tmfset** [odevices/er-301#78] [General]
Hey @odevices I was thinking about adding a new feature to the lua unit where you can set a background for a view (control group). In my particular case I want to show the sample view at all times behind the main unit controls. Here's a proof of concept I was able to hack up:

![0004](https://user-images.githubusercontent.com/69446/214683603-673894df-53a6-4090-bbd5-b171fbb85d78.png)

Originally I was going to re-implement a large portion of the view control layer to get this working but after some more reading I realized the SectionView can be modified to simply add a `background` graphic that gets rendered in when the view is rebuilt.

The UI in this example clearly still needs work to de-clutter, but I think this would be a powerful simple change that opens up a lot of doors for more experimental interfaces.

Edit: here's a quick gif to demonstrate better
![sloop-view-background](https://user-images.githubusercontent.com/69446/214685874-f2da65fd-144e-4a17-97fd-dec14286f5e6.gif)

**Comment by tmfset** [comment]
I've made a PR with the relevant changes here, https://github.com/odevices/er-301/pull/79

**Reply by teddarcuri** [reply]
This is awesome! I am new to the 301 and haven't rolled up my sleeves and gotten into the sdk yet, but is there a way to adjust opacity? even just making the background less opaque might do the trick. I dig this idea though!
